cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -140,6 +137,17 @@ def is_displayed(self):
         )
 
 
+class AddOrchestrationCatalogItemView(AddCatalogItemView):
+    fill_strategy = WaitFillViewStrategy()
+
+    @property
+    def widget_names(self):
+        return ['title', 'select_catalog', 'select_dialog',
```

Comment:
Please put inline comment here that you're setting widget_names in order to control the fill order of this view, but the widgets are the same as AddCatalogItemView